lib/components.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const path = require('path');
+const log = require('./logger')('nodecg/lib/components');
+const Bundles = require('./bundle-manager');
+
+log.trace('Adding Express routes for bundle components');
+
+const all = Bundles.all();
+
+for (let i = 0; i < all.length; i++) {
```

Comment:
I usually prefer to use `<array>.forEach` instead of `for` loops, where possible.